react/components/Table/SimpleTable.js

```diff
@@ -56,7 +56,7 @@ class SimpleTable extends Component {
   }
 
   handleRowHover = rowIndex => {
-    const { onRowClick } = this.props
+    const { onRowClick, onRowHover = () => {} } = this.props
```

Comment:
shouldn't this be a default prop instead of a default paramater?

Reply:
Yes! Fixed it